assets/js/modules/analytics-4/components/settings/OptionalSettingsView.js

```diff
@@ -48,8 +44,6 @@ export default function OptionalSettingsView() {
 		select( MODULES_ANALYTICS_4 ).getAdsConversionID()
 	);
 
-	const showAdsConversionIDSettings = canUseSnippet && useSnippet;
```

Comment:
Just a note for reference that this change is intentional and is done based on conversation between Jimmy and myself. We will soon remove the `canUseSnippet` module setting and there is a chance leaving it here may cause unexpected issues as described [here](https://github.com/google/site-kit-wp/pull/8231#discussion_r1482361990).